main.go

```diff
@@ -67,7 +66,8 @@ func run(c *cli.Context) error {
 	var G, fullGraph *Graph
 
 	if c.IsSet("file") && c.IsSet("puppet") {
-		log.Fatal("the --file and --puppet parameters cannot be used together")
+		log.Println("the --file and --puppet parameters cannot be used together")
+		return cli.NewExitError("invalid parameters", 1)
```

Comment:
Remove the "invalid parameters" text and leave an empty string, the log function spits out the nice message with a timestamp, where as the NewExitError function doesn't log it. This way it will be consistent throughout. (Run it to see what I mean.)